app/src/main/java/com/example/playlistmaker/App.kt

```diff
@@ -22,15 +24,15 @@ class App : Application() {
         super.onCreate()
         PermissionRequester.initialize(this)
         startKoin {
-            androidLogger()
             androidContext(this@App)
             modules(
                 findModule,
                 mediaPlayerModule,
                 settingsModule,
                 sharingModule,
                 mediaLibraryModule,
-                newPlaylistModule
+                newPlaylistModule,
+                playlistModule
```

Comment:
nit: можно воспользоваться [include и указывать в App только один родительский модуль](https://insert-koin.io/docs/reference/koin-android/modules-android/#module-includes-since-32)